bin/Makefile.am

```diff
@@ -22,15 +22,11 @@ MAINTAINERCLEANFILES = \
 	$(NULL)
 
 EXTRA_DIST = \
-	ovirt-ha-agent.in \
-	ovirt-ha-broker.in \
 	$(NULL)
 
 scriptsdir = $(engine_ha_bindir)
 
 dist_scripts_SCRIPTS = \
-	ovirt-ha-agent \
```

Comment:
This should be kept, as this makes copies the files in the correct folder on make dist.